pyproject.toml

```diff
@@ -5,9 +5,9 @@ description = "Nautobot Floor Plan"
 authors = ["Network to Code, LLC <info@networktocode.com>"]
 license = "Apache-2.0"
 readme = "README.md"
-homepage = "https://github.com/nautobot/nautobot-plugin-floor-plan"
-repository = "https://github.com/nautobot/nautobot-plugin-floor-plan"
-keywords = ["nautobot", "nautobot-plugin"]
+homepage = "https://github.com/nautobot/nautobot-app-floor-plan"
```

Comment:
Would it be appropriate or desirable to change the `homepage` to point to the docs instead of the GitHub repo?

Reply:
I would say yes, this should go to the cookie first